### PR TITLE
Dyn 3070 coverage python node models wpf 2

### PR DESCRIPTION
**Pull request by RobertGlobant20**
### Purpose
I added tests for code covering the next methods in the PythonNodeModelsWpf\PythonNode.cs file.

- OpenPythonLearningMaterial(object sender, RoutedEventArgs e)
- EditScriptContent(object sender, EventArgs e)

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [X] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@QilongTang 

### FYIs

@alfredo-pozo

### CODE REVIEW

File: test/DynamoCoreWpfTests/PythonNodeCustomizationTests.cs [hunk 1]
```diff
@@ -270,6 +270,155 @@ public void OnMoreInfoEventTest()
...
+
+            //Get the python engine menu 
+            var engineMenuItem = nodeView.MainContextMenu
+                .Items
+                .Cast<MenuItem>()
+                .First(x => x.Header.ToString() == "Python Engine Version");
+
+            Assert.IsNotNull(engineMenuItem);
+
+            //Get the python engime menu CPython3
```
Comment (QilongTang) [inline test/DynamoCoreWpfTests/PythonNodeCustomizationTests.cs:368]: typo `engine`

**Review by QilongTang — APPROVED** [review]
LGTM with one typo to fix

File: test/DynamoCoreWpfTests/PythonNodeCustomizationTests.cs (hunk 1 above)
Comment (RobertGlobant20) [reply test/DynamoCoreWpfTests/PythonNodeCustomizationTests.cs]: @QilongTang fixed in the commit.
https://github.com/DynamoDS/Dynamo/pull/11156/commits/0036fd5fff2814eb944d45af4885f9242fd444e8

**Comment by QilongTang** [comment]
The regressions on selfServe are known issues. Merging

**Pull request merged**